src/main/java/com/system/billingsystem/entities/InvoiceVoucher.java

```diff
@@ -1,5 +1,8 @@
-package com.system.billingSystem.entities;
+package com.system.billingsystem.entities;
 
+import org.graalvm.nativeimage.c.constant.CEnum;
```

Comment:
What is this for?

Reply:
With out this Spring Boot does not read this enum and throw the error "simbol not find". The same problem apare on BaseRepository with out "@Repository".